src/commands/blocks.ts

```
@@ -290,7 +294,18 @@
 				}),
 			};
 
-			log(JSON.stringify(jsonOutput, null, 2));
+			// Process with jq if specified
+			if (ctx.values.jq != null) {
```

ESLint

[error] Unsafe member access .values on an `any` value.

(ts/no-unsafe-member-access)